readme.md

```diff
@@ -0,0 +1,150 @@
+# Native RDF loaders for the [code](https://code.described.at) vocabulary
+
+The loaders contained in this packages let developers link RDF to native code and load that code at runtime
+to be exacuted in their programs.
+
+## Installation
+
+Currently must be installed from git
+
+```
+npm i --save https://github.com/zazuko/rdf-native-loader-code
+npm i --save https://github.com/zazuko/rdf-native-loader
+```
+
+Somewhere at the beginning of your code, register the loaders
+
+```js
+const LoaderRegistry = require('rdf-native-loader')
+const EcmaScriptLoader = require('rdf-native-loader-code/ecmaScript')
+const EcmaScriptLiteralLoader = require('rdf-native-loader-code/ecmaScriptLiteral')
+
+const registry = new LoaderRegistry()
+
+EcmaScriptLoader.register(registry)
+EcmaScriptLiteralLoader.register(registry)
+
+module.exports = registry
+```
+
+## Usage
+
+All examples below assume that the `./dataset` module exports a [`RDF/JS` dataset](https://github.com/rdfjs/dataset-spec).
+The `./registry` module is assumed to be implemented as shown in the `Installation` section above.
+
+### JS code loaded from node modules (or built-ins)
+
+This example is equivalent to calling `require('fs').createReadStream` 
+
+```turtle
+@prefix code: <https://code.described.at/>
+
+<urn:example:node> 
+  code:implementedBy [
+    a code:EcmaScript ;
+    code:link <node:fs#createReadStream>
+  ] .
+```
+
+```js
+const cf = require('clownface')
+const rdf = require('rdf-ext')
+const registry = require('./registry')
+const dataset = require('./dataset')
+
+const parentNode = rdf.namedNode('urn:example:node')
+const implementedBy = rdf.namedNode('https://code.described.at/implementedBy')
+
+const createReadStream = registry.load(cf(dataset).node(parentNode).out(implementedBy))
+```
```

Comment:
I think it would be good to show how these examples can be run. For instance here it is not clear to me what to do with the turtle file.

Reply:
I tried to explain that right under `## Usage`. The assumption is that the `require('./dataset')` parser the graph and exports a RDFJS dataset.

Reply:
I see, I don't think I could have guessed by reading the usage section. I think the issue is that *usage* says there should be a `./dataset` module exporting a dataset but there's no word about what this dataset should contain. What about adding a sentence there to make it clear that this dataset is the pipeline data as parsed from the turtle examples?

Reply:
Sure @vhf. Check what I added

Reply:
Perfect, thanks!